index.js

```diff
@@ -1,6 +1,13 @@
 var postcss = require('postcss')
+var postcssrc = require('postcss-load-config')
```

Comment:
The current package supports `nodejs 8` and higher, we can safely use `const` he is holding on `nodejs` 6 and up

Reply:
I didn't want to pollute the diff by changing `var`s to `const`s etc.

Reply:
I understand, don't worry about it)